core/database/foxx/tests/user_fixture.js

```diff
@@ -0,0 +1,24 @@
+const { db } = require("@arangodb");
+const g_lib = require("../api/support");
+
+const uid = "testUser0";
```

Comment:
It would be nice to also export this and make it available to tests. I'm not sure how I want to do that right now, and I think it's out of scope. Just a thought.

Reply:
Right now yes, that being said it will definately be on the agenda as QA when we're wrapping up this work.